sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/Customized/Models/RequestReportRecordContract.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+using System.Net;
+using Azure.Core;
+
+namespace Azure.ResourceManager.ApiManagement.Models
+{
+    /// <summary> Request Report data. </summary>
+    public partial class RequestReportRecordContract
+    {
+        /// <summary> The HTTP status code received by the gateway as a result of forwarding this request to the backend. </summary>
+        [EditorBrowsableAttribute(EditorBrowsableState.Never)]
+        [ObsoleteAttribute("This property has been replaced by BackendResponseCodeInteger", false)]
```

Comment:
minor: could be simplified to `[Obsolete("This property has been replaced by BackendResponseCodeInteger", false)]`

Reply:
Resolved